pylint/lint/parallel.py

```diff
@@ -62,9 +63,13 @@ def _worker_initialize(linter, arguments=None):
     _patch_sys_path(arguments or ())
 
 
-def _worker_check_single_file(file_item):
+def _worker_check_single_file(
+    file_item: Tuple[str, str, str]
+) -> Tuple[int, Any, str, Any, List[Tuple[Any, ...]], Any, Any, DefaultDict[Any, List]]:
     name, filepath, modname = file_item
 
+    if not _worker_linter:
+        raise Exception("Worker linter not yet initialised")
```

Comment:
`mpy` complains on the following two lines about the possibility that `_work_linter` might be None.
This is probably the wrong `Exception` type.
We currently already test for this, but do so with catching an `AttributeError` on L73. See:
https://github.com/PyCQA/pylint/blob/fab2de272b3fc36dc4480d89840b8bfe0fbfbd0c/tests/test_check_parallel.py#L177-L182
I think making this error more explicit as `mypy` wants makes sense.